### ISSUE TITLE
Library WebUI

### ISSUE THREAD

**Issue by gbougakov** [npm/npm#17766]
#### What's the feature?
WebUI for library management
#### What problem is the feature intended to solve?
IDK
#### Is the absence of this feature blocking you or your team? If so, how?
No
#### Is this feature similar to an existing feature in another tool?
No
#### Is this a feature you're prepared to implement, with support from the npm CLI team?
Yes

**Comment by kenany** [comment]
I'm not really sure what you are asking for here. Are you requesting some change be made to npmjs.com?

**Comment by gbougakov** [comment]
@KenanY no. I want a local interface to manage projects on your box (machine/computer/server/etc), dependencies, updates and everything else you can do in CLI

**Comment by gbougakov** [comment]
@KenanY I'd like to implement a feature, that starts your browser when you run `npm ui` command.
In browser, you will see an UI where you can manage dependencies or other stuff
